FrontEnd/ProfilePage.html

```diff
@@ -3,8 +3,43 @@
 <head>
     <meta charset="UTF-8">
     <title>Title</title>
+    <link rel="stylesheet" href="css/ProfilePage.css">
 </head>
 <body>
 
+    <div id="exit_div">
+        <p>Hit Esc to go back</p>
+        <img src="images/exit_button.png" alt="escape">
+    </div>
+
+    <div id="myaccount"> <p>My Account</p> </div>
+
+    <main>
+        <div id="banner">
+        <!-- Some images will go here eventually -->
+
+            <img src="images/black.jpg" alt="profile_pic" id="profile_pic">
+        </div>
+        <div id="information">
+            <div id="name">
+                <p>Name goes here <button type="button">Edit Profile</button></p>
+            </div>
+        <!-- This is for showing the actual information-->
+            <div id="more_info">
+                <div class="user_information">
+                    <p><b>DISPLAY NAME: </b> Goes here <button type="button">Edit</button></p>
```

Comment:
Please provide a short comment on how these buttons will be implemented using javascript